cdc/sink/mysql.go

```diff
@@ -74,6 +74,9 @@ var (
 	}
 )
 
+type mysqlSyncpointSink struct {
```

Comment:
the `mysqlSyncpointSink` is not a sink now, the `xxxSink` is no longer  suitable